examples/hybrid/box/limiters_advection.jl

```diff
@@ -373,8 +317,7 @@ for (k, horz_ne) in enumerate(horz_ne_seq)
     ystar = copy(y0)
     parameters = (
         horzspace = horzspace,
-        min_q = min_q,
-        max_q = max_q,
+        limiter = Limiters.QuasiMonotoneLimiter(y0.ρq, y0.ρ),
```

Comment:
Thank you for this much nicer interface! Just a question. I know here `y0` is copied into `ystar` anyway, but to avoid any confusion on the variable names, shouldn't we pass `(ystar.ρq, ystar.ρ)` to the limiter function so that it doesn't look like we would be limiting the initial condition only?

Reply:
Also, can you advice/show how this interface would be used in a problem that has multiple tracers, e.g., in #702?

Reply:
we could, this is just used to set up the state, it doesn't matter which one we use.

Reply:
I know, but given the confusion on the state Vs tendency that was brought up the other day, I'd try to avoid these types of things. 

Also, can you please show/advise on how to use this for an arbitrary number of tracers? Via a variable tuple?